gateways/orbital/request_builders.go

```diff
@@ -0,0 +1,84 @@
+package orbital
+
+import (
+	"encoding/xml"
+	"strconv"
+
+	"github.com/BoltApp/sleet"
+)
+
+func buildAuthRequest(authRequest *sleet.AuthorizationRequest) Request {
+
+	amount := authRequest.Amount.Amount
+	exp := strconv.Itoa(authRequest.CreditCard.ExpirationYear) + strconv.Itoa(authRequest.CreditCard.ExpirationMonth)
+	code := currencyMap[authRequest.Amount.Currency]
+
+	body := RequestBody{
+		IndustryType:     IndustryTypeEcomm,
+		MessageType:      MessageTypeAuth,
+		BIN:              BINStratus,
+		TerminalID:       TerminalIDStratus,
```

Comment:
Are we just assuming BIN and Terminal ID are Stratus for simplicity sake currently?

Reply:
Yes, not sure what the logic would be for determining when to use "Stratus" vs "Tandem"

Reply:
Ya I think it would have to be an additional value we accept. I think most merchants for card-not-present will be Stratus, so we can just worry about it if there's ever a merchant that isn't Stratus